src/js/common/schemaform/actions.js

```diff
@@ -183,7 +183,6 @@ export function uploadFile(file, filePath, uiOptions, progressCallback) {
         if (req.status >= 200 && req.status < 300) {
           const body = 'response' in req ? req.response : req.responseText;
           const fileData = uiOptions.parseResponse(JSON.parse(body), file);
-          fileData.file = file;
```

Comment:
It looks like a test set up for the [screen reader changes ](https://github.com/department-of-veterans-affairs/vets-website/commit/98e1324c2ad872e04eae2565c619047bad6a4a80)needs to be removed as well.

https://github.com/department-of-veterans-affairs/vets-website/blob/c089c56c6e4b6893df3c1df9febb0ab48e7b8963/test/common/schemaform/actions.unit.spec.js#L362-L365